### PR TITLE
Update mrl parsing

### PR DESCRIPTION
**Pull request by HenryOfCarim**
.ksy format now is almost reversed, and import was tweaked to match with current findings

### CODE REVIEW

File: albam/engines/mtfw/structs/mrl.ksy [hunk 1]
```diff
@@ -18,36 +19,131 @@ seq:
...
+        type: 
+          switch-on: shader_object_hash
+          cases:
+            0x7b2c215f: shd_s_globals # $Globals re0
+            0x6c801200: shd_cb_material # CBMaterial re0
+            0x7b2c2159: shd_s_globals # $Globals rehd
+            0x6c8011f9: shd_cb_material # CBMaterial rehd
+            0x7b2c2155: shd_s_globals # $Globals rer1
+            0x6c8011f4: shd_cb_material # CBMaterial rer1
+            0x7b2c215e: shd_s_globals # $Globals rer2
```
Comment (HenryOfCarim) [inline albam/engines/mtfw/structs/mrl.ksy:45]: I thought in all games same commands for increasing the float buffer have the same size but in the case of ReRev2 $Globals size is bigger than 76 floats

Comment (Brachi) [reply albam/engines/mtfw/structs/mrl.ksy:45]: I'll run tests to verify it. I think we could add an `app_id` parameter to decide on size as well.

**Review by Brachi — APPROVED** [review]
Great work!

File: albam/engines/mtfw/structs/mrl.ksy (hunk 1 above)
Comment (Brachi) [reply albam/engines/mtfw/structs/mrl.ksy]: minimal tests for mrl pushed. Will improve tests later.

**Pull request merged**